components/header.js

```diff
@@ -52,7 +55,20 @@ export default function Header ({ sub }) {
     {
       hasNewNotes
     }
-  `, { pollInterval: 30000, fetchPolicy: 'cache-and-network' })
+  `, {
+    pollInterval: 30000,
+    fetchPolicy: 'cache-and-network',
+    // Trigger onComplete after every poll
+    // See https://github.com/apollographql/apollo-client/issues/5531#issuecomment-568235629
+    notifyOnNetworkStatusChange: true,
+    onCompleted: (data) => {
+      const notified = JSON.parse(localStorage.getItem('notified')) || false
+      if (!notified && data.hasNewNotes) {
+        notification.show('you have new notifications')
+      }
+      localStorage.setItem('notified', data.hasNewNotes)
```

Comment:
When is this cleared?

Reply:
Never mind. It's cleared on the next poll. Duh

Reply:
yeah, all i want with this is to see the "rising edge" when we move from "no new notes" to "new notes".
not sure if this is the best way